practice/practice-02/src/includes/mixins.pug

```diff
@@ -0,0 +1,25 @@
+- var values = [1, 2, 3];
+
+mixin cardFoodItem(itemFood, price, ratePoint, foodName, foodDescription)
+  .card-item
+    img.card-img(src=offer_img_path + "item-card.png", alt="Item card image")
+    img.food-image(
+      src=offer_img_path + "food-" + itemFood + ".png",
+      alt="Food " + itemFood + " image"
+    )
+    span.price= price + "$"
+    .content-card
+      .rate
+        .avatar-container
+          each val in values 
+            img(
+              src=offer_img_path + "avatar-" + val + "-item-" + itemFood + ".png",
+              alt="Avatar user rate",
+              class="avatar-" + val
+            )
+        .star-container
+          img(src=offer_img_path + "star.png", alt="Star image")
+        span.rate-point= "(" + ratePoint + ")"
+      span.food-name= foodName
```

Comment:
Why name is span, but description is p?

Reply:
Because I think the p element should be used for paragraphs, in this case, should not use for food name. I can't think of anything better than that. Can you suggest the element for this?